zips/zip-0321.rst

```diff
@@ -45,15 +45,15 @@ Motivation
 ==========
 
 In order for a robust transactional ecosystem to evolve for Zcash, it is
-necessary for vendors to be able to issue requests for payment. At present, the
-best option available is to manually specify a payment address, a payment
-amount, and potentially memo field content. Of these three components, existing
-wallets only provide functionality for reading payment addresses in a
-semi-automated fashion. It is then necessary for the user to manually enter
-payment amounts and any associated memo information, which is tedious and may
-be error-prone, particularly if a payment is intended for multiple recipients
-or the memo field information contains structured data that must be faithfully
-reproduced.
+necessary for vendors to be able to issue requests for payment. At present,
+the best option available is to manually specify a payment address, a payment
+amount, and potentially asset id and memo field content. Of these three
+components, existing wallets only provide functionality for reading payment
+addresses in a semi-automated fashion. It is then necessary for the user to
+manually enter payment asset ids, amounts and any associated memo information,
+which is tedious and may be error-prone, particularly if a payment is intended
+for multiple recipients or the memo field information contains structured data
+that must be faithfully reproduced.
```

Comment:
This change to the motivation doesn't make sense, because asset IDs did not exist when the ZIP was written (and the "Of these three" components becomes incorrect after the edit). Instead, change "At present" to "At time of first writing this ZIP".